app/components/ui/files/publish-modal/template.hbs

```diff
@@ -129,21 +133,21 @@
     {{/ui-dropdown}}
     
     {{#if enablePublish}}
-    <div class="ui positive blue enabled right labeled icon button">
+    <div class="ui positive blue enabled right labeled icon button" style="position:absolute; right:0px;" {{action "publishedClicked"}}>
```

Comment:
@ThomasThelen instead of adding the inline style, better create a new class or mixin with those styles you need, and add it to the existing ones `ui positive blue enabled right labeled icon button`